nodejs/week1/avg.js

```diff
@@ -0,0 +1,12 @@
+const inputNumbers = process.argv.slice(2).map(Number)
+// slice(2), 2 because 0 and 1 index takes service information
+
+//check that the entered number is a number.
+for (let i = 0; i < inputNumbers.length; i++) {
+  if (isNaN(inputNumbers[i])) {
+    console.log(`${process.argv[i + 2]} is not a number!`)
```

Comment:
If this check becomes true we usually do not want to continue running the rest of the code. 
You can use [process.exit ](https://nodejs.org/api/process.html#processexitcode)to instruct Node.js to exit the process.  
This way we do not try to do the calculation if we know that the input is not valid.

 ```suggestion
    console.log(`${process.argv[i + 2]} is not a number!`)
    process.exit(1)
```
In this context the side effects are not too bad. We will simply get `NaN` printed to the terminal. However, in other cases we need to be careful to handle errors gracefully.